### DISCUSSION TITLE
I can't figure out how to get NFT's after completing this section (3).

### DISCUSSION THREAD

**Discussion by Rayyan-Uddin** [Cyfrin/foundry-full-course-cu#2370] [General]
https://explorer.zksync.io/address/0x598c64586d890222c085eCb3a01Db36237D6ccE4

https://sepolia.etherscan.io/address/0x1b30DA2a868704483143A4D46865Ac9585629fd0#code

After opening these above links I just don't know what to do next, HELP.

**Comment by EngrPips** [comment]
Hello @Rayyan-Uddin, Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2274

**Comment by Rayyan-Uddin** [comment]
Thanks, Appreciated.

**Reply by EngrPips** [reply]
Anytime, Friend.
